### DISCUSSION TITLE
Suggestion: Start with last screen

### DISCUSSION THREAD

**Discussion by HubKing** [UnicornsOnLSD/finamp#557] [General]
It seems that it always starts with the Album listing. I think it would be more convenient if the last screen, such as the album page or the song screen was restored.

**Comment by bmyoungquist** [comment]
I would absolutely love this feature! I may take a look at the code after the holiday to see if I can figure this one out.

**Reply by Chaphasilor** [reply]
It should be fairly easy to implement. You can take a look at https://github.com/jmshrv/finamp/commit/61683c63afaa4436d9ac672ab15e5f93deb9aea5 to see how it could be done.

Also, make sure to read the contribution guidelines, as there is one command that you will need to run after making changes to the Hive stuff, and it's explained in there :)

**Comment by jonabatero** [comment]
Greetings, I just installed this app because I was looking for something more focused on music within the jellyfin ecosystem.
In general terms it is very good. But I would like to provide some comments that can be taken into account:

 - native DSD file support (transcoding is good but format and codec information is not transparent)

- be able to see all the information about the tracks from the app (technical information about the formats, etc.)

With these 2 points incorporated I think it would be an option identical to what Plexamp offers.

greetings.
